drivers/hmis/app/models/hmis/hud/processors/client_processor.rb

```diff
@@ -50,6 +50,9 @@ def process(field, value)
       when 'veteran_status'
         # Veteran status is non-nullable. It should be saved as 99 even if hidden. (It's hidden for minors)
         { attribute_name => attribute_value || 99 }
+      when 'age_range'
+        # Prioritize exact DOB if it is provided
+        process_age_range(value) unless @hud_values.key?('Client.dob') && @hud_values['Client.dob'].present?
```

Comment:
I don't think the first `key?` condition is needed. This works fine: `{}['key'].present?`